lms/product/plugin/grouping_service.py

```diff
@@ -62,10 +62,6 @@ def get_groups_for_grading(
 
         return None
 
-    @classmethod
-    def from_request(cls, request):
-        return cls()
```

Comment:
If a class requires no special init, we can now create it as a singleton and register it once for the whole app.